packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.tsx

```diff
@@ -13,13 +13,40 @@ export class SwirlTableRowGroup {
 
   @Prop() label!: string;
   @Prop() tooltip?: string;
+  @Prop() collapsable?: boolean = false;
+
+  @State() isVisible: boolean = true;
+
+  private toggleShowItems = () => {
+    const rowsContainer = this.el.shadowRoot.querySelector(
+      ".rows-container"
+    ) as HTMLElement;
+    if (this.isVisible) {
+      rowsContainer.style.height = "0";
+    } else {
+      rowsContainer.style.height = "auto";
+      const height = rowsContainer.clientHeight + "px";
+      rowsContainer.style.height = "0";
+      setTimeout(() => (rowsContainer.style.height = height), 0);
+    }
+    this.isVisible = !this.isVisible;
+  };
 
   render() {
     const rowspan = this.el.querySelectorAll("swirl-table-row").length;
+    const Icon = this.isVisible
+      ? "swirl-icon-expand-less"
+      : "swirl-icon-expand-more";
 
     return (
       <Host class="table-row-group" role="rowgroup">
         <div class="table-row-group__header-row" role="row">
+          {this.collapsable && (
+            <Icon
```

Comment:
Suggestion: Would be great if you could implement the Disclosure pattern here. It's pretty straight forward. Just an aria-expanded on the button. https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/

Reply:
I see! I added it. Cool, thank you for the resources!!